fdbserver/VersionedBTree.actor.cpp

```diff
@@ -5291,12 +5440,10 @@ class VersionedBTree {
 
 			// If we are only writing 1 page and it has the same BTreePageID size as the original then try to reuse the
 			// LogicalPageIDs in previousID and try to update them atomically.
-			if (pagesToBuild.size() == 1 && previousID.size() == pages.size()) {
-				for (k = 0; k < pages.size(); ++k) {
-					LogicalPageID id = wait(
-					    self->m_pager->atomicUpdatePage(PagerEventReasons::Commit, height, previousID[k], pages[k], v));
-					childPageID.push_back(records.arena(), id);
-				}
+			if (pagesToBuild.size() == 1 && previousID.size() == p.blockSize && p.blockSize == 1) {
```

Comment:
This is a bug, `blockSize` should be `blockCount`.

Reply:
Just curious if this was caught by a test and if not, could a test (unit or simulation) have caught it?

Reply:
Code coverage via `TEST()` macros would catch it.  I'll have those in soon.